templates/.editorconfig.tpl

```diff
@@ -12,6 +12,17 @@ indent_style = tab
 indent_style = space
 indent_size  = 2
 
+[*.py]
+indent_style = space
+indent_size = 4
+
+[*.{json,toml}]
+indent_style = space
+indent_size = 2
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
question: what is the rationale for this?

Reply:
In Markdown syntax:
- Trailing spaces at the end of a line have semantic meaning.
- Two or more spaces at the end of a line produce a line break (<br> in HTML output).